ui/lib/kubernetes/addon/routes/roles/role/credentials.js

```diff
@@ -1,3 +1,12 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+export default class KubernetesRoleCredentialsRoute extends Route {
+  @service secretMountPath;
 
-export default class KubernetesRoleCredentialsRoute extends Route {}
+  model() {
+    return {
+      roleModel: this.modelFor('roles.role'),
+      backend: this.secretMountPath.get(),
```

Comment:
I think this could be simplified to where we wouldn't need the `secretMountPath` service. Since we have the role model available which already has the backend value in it we could just return the role model here and access the backend value on the role from it.
```js
model() {
  return this.modelFor('roles.role');
}
```